// @awesome.me/webawesome/dist-cdn/webawesome.loader.js
/*! Copyright 2026 Fonticons, Inc. - https://webawesome.com/license */
import "./chunks/chunk.4IALDZ4Y.js";
import "./chunks/chunk.D5YFE5NT.js";
import "./chunks/chunk.N2TXQSKF.js";
import {
  serialize
} from "./chunks/chunk.LRYJ2M5H.js";
import "./chunks/chunk.KBS6YHTA.js";
import {
  discover,
  preventTurboFouce,
  startLoader,
  stopLoader
} from "./chunks/chunk.ZFSRFTCP.js";
import {
  allDefined
} from "./chunks/chunk.X55YNZ3B.js";
import {
  getAnimationNames,
  getEasingNames
} from "./chunks/chunk.LN7M2NWC.js";
import "./chunks/chunk.WQ2OSH23.js";
import "./chunks/chunk.MFUK3XY6.js";
import {
  registerTranslation
} from "./chunks/chunk.CZ2YL77F.js";
import {
  getDefaultIconFamily,
  registerIconLibrary,
  setDefaultIconFamily,
  unregisterIconLibrary
} from "./chunks/chunk.NF5JTFKH.js";
import {
  getIconFolder
} from "./chunks/chunk.HCXBOJYW.js";
import {
  getBasePath,
  getIconPath,
  getKitCode,
  setBasePath,
  setIconPath,
  setKitCode
} from "./chunks/chunk.HGBRCPUS.js";
import "./chunks/chunk.XTA2JDH4.js";
import "./chunks/chunk.JHZRD2LV.js";
export {
  allDefined,
  discover,
  getAnimationNames,
  getBasePath,
  getDefaultIconFamily,
  getEasingNames,
  getIconFolder,
  getIconPath,
  getKitCode,
  preventTurboFouce,
  registerIconLibrary,
  registerTranslation,
  serialize,
  setBasePath,
  setDefaultIconFamily,
  setIconPath,
  setKitCode,
  startLoader,
  stopLoader,
  unregisterIconLibrary
};
